legal-api/src/legal_api/resources/v2/document.py

```diff
@@ -0,0 +1,52 @@
+# Copyright © 2024 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Module for handling Minio document operations."""
+
+from http import HTTPStatus
+
+from flask import Blueprint, current_app, jsonify
+from flask_cors import cross_origin
+
+from legal_api.models import Document, Filing
+from legal_api.services.minio import MinioService
+from legal_api.utils.auth import jwt
+
+
+bp = Blueprint('minio', __name__, url_prefix='/api/v2/documents')
+
+
+def is_draft_filing(file_key: str) -> bool:
+    """Check if the filing is a draft filing."""
+    document = Document.query.filter_by(filing_id=file_key).first()
```

Comment:
create new function in Document model to fetch with file_key instead of querying here